graphblas_algorithms/conftest.py

```diff
@@ -42,3 +55,15 @@ def orig():
             if getattr(module, key, None) is orig_val:
                 setattr(module, key, new_val)
     yield orig
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ic():
```

Comment:
Remind me to show this off in our next meeting!  See:
https://github.com/gruns/icecream